### DISCUSSION TITLE
How to check wal2json installed

### DISCUSSION THREAD

**Discussion by aironyiu** [eulerto/wal2json#234] [Q&A] [answered]
Hi
In our production EDB PG environment, one previous DBA installed a version of wal2json for CDC software. 
Is there a way to find out the version of wal2json installed and configured so that we can install the same version on new servers.

Thks

**Comment by eulerto** [comment, marked as answer]
wal2json does not provide a control file so the installed version is **not** available in the `pg_available_extensions` view. How did he/she install it? Package? Check the package version in your OS. If he/she used a source code, you have to inspect the build source tree.

**Comment by aironyiu** [comment]
Thks.

取得 Android 版 Outlook<https://aka.ms/AAb9ysg>
________________________________
